images/devtools-kubernetes-v1beta1/context/Dockerfile

```diff
@@ -1,92 +1,79 @@
+COPY --from=buildenv /go/bin/dyff /usr/local/bin/dyff
+
+COPY ./magefiles /magefiles
+#RUN \
+#    mkdir -vp ~/.cache ~/.local/bin ~/bin && \
+#    true
+#
+#
+#COPY maker /usr/local/bin/maker
+#RUN \
+#    ln -rs /usr/local/bin/maker /usr/local/bin/validate && \
+#    true
 
-RUN \
-    mkdir -vp ~/.cache ~/.local/bin ~/bin && \
-    true
-
-ARG kubeval_version="0.16.1"
-COPY kubeval_${kubeval_version}_checksums.txt /var/tmp/build/kubeval_${kubeval_version}_checksums.txt
 
-# Kubeval only publishes a docker image with a binary that is dynamically linked musl libc, download release artifact directly
-RUN \
-    cd /var/tmp/build && \
-    . /usr/local/bin/goenv.sh && \
-    curl -LOJ "https://github.com/instrumenta/kubeval/releases/download/v${kubeval_version}/kubeval-${GOOS}-${GOARCH}.tar.gz" && \
-    sed -i "/-${GOOS}-${GOARCH}.tar.gz/!d" "kubeval_${kubeval_version}_checksums.txt" && \
-    sha256sum -c "kubeval_${kubeval_version}_checksums.txt" && \
-    tar -xvf "kubeval-${GOOS}-${GOARCH}.tar.gz" -C /usr/local/bin kubeval && \
-    kubeval --version && \
-    true
-
-ARG kubectl_version="1.23.0"
-COPY kubectl_${kubectl_version}_checksums.txt /var/tmp/build/kubectl_${kubectl_version}_checksums.txt
-
-# From https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/
-# Use ./get_kubectl_checksums to retrieve checksums for a release
-RUN \
-    cd /var/tmp/build && \
-    . /usr/local/bin/goenv.sh && \
-    curl -LJ "https://dl.k8s.io/release/v${kubectl_version}/bin/${GOOS}/${GOARCH}/kubectl" -o "kubectl-${GOOS}-${GOARCH}" && \
-    sed -i "/-${GOOS}-${GOARCH}/!d" "kubectl_${kubectl_version}_checksums.txt" && \
-    sha256sum -c "kubectl_${kubectl_version}_checksums.txt" && \
-    install -o root -g root -m 0755 "kubectl-${GOOS}-${GOARCH}" /usr/local/bin/kubectl && \
-    kubectl version --client=true && \
-    true
-
-ARG kubeconform_version="0.4.13"
-COPY kubeconform_${kubeconform_version}_checksums.txt /var/tmp/build/kubeconform_${kubeconform_version}_checksums.txt
-RUN \
-    cd /var/tmp/build && \
-    . /usr/local/bin/goenv.sh && \
-    curl -LOJ "https://github.com/yannh/kubeconform/releases/download/v${kubeconform_version}/kubeconform-${GOOS}-${GOARCH}.tar.gz" && \
-    sed -i "/-${GOOS}-${GOARCH}.tar.gz/!d" "kubeconform_${kubeconform_version}_checksums.txt" && \
-    sha256sum -c "kubeconform_${kubeconform_version}_checksums.txt" && \
-    tar -xvf "kubeconform-${GOOS}-${GOARCH}.tar.gz" -C /usr/local/bin kubeconform && \
-    kubeconform -v && \
-    true
-
-RUN \
-    rm -rf /var/tmp/build && \
-    true
-
-COPY maker /usr/local/bin/maker
-RUN \
-    ln -rs /usr/local/bin/maker /usr/local/bin/help && \
-    ln -rs /usr/local/bin/maker /usr/local/bin/validate && \
-    true
+VOLUME /srv/workspace
+WORKDIR /srv/workspace
 
-COPY Makefile /usr/local/share/devtools-kubernetes/Makefile
 
 RUN git config --system --add safe.directory /srv/workspace
 
+ENTRYPOINT ["/usr/local/bin/mage-k8s"]
```

Comment:
No, you can modify entrypoint on run

Reply:
as noted in PR Body

> I also changed the entry point to be the complied mage binary instead of being bash, but i feel that the user experience is just better like this mainly because it is now easy to discover which targets there are.